backend/WP/Gutenberg/js/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "devDependencies": {
+    "@babel/core": "^7.5.5",
+    "@babel/preset-react": "^7.0.0",
+    "@wordpress/babel-plugin-import-jsx-pragma": "^2.2.0",
```

Comment:
From the documentation https://developer.wordpress.org/block-editor/packages/packages-element/, I think this plugin and its configuration should be set up when using directly the package @wordpress/element. 
If that's the case, then it must have the correct configuration https://developer.wordpress.org/block-editor/packages/packages-babel-plugin-import-jsx-pragma/.